db/db_impl/db_impl.cc

```diff
@@ -472,14 +472,16 @@ Status DBImpl::ResumeImpl(DBRecoverContext context) {
     // FlushAllColumnFamilies releases and re-acquires mutex.
     if (shutdown_initiated_) {
       s = Status::ShutdownInProgress();
-    } else {
```

Comment:
`} else if (context.flush_after_recovery) {`

would do the same without re-checking s.ok() below, but not a big deal.

Reply:
Do you mean scheduling compaction in the else if block? We need to do that when context.flush_after_recovery is false too.

Reply:
Ahh never mind. I missed the collapsed part of the code. I didn't see we were calling FlushAll() above between line 460 - 471

I originally meant line 456 would stay as if (s.ok()), but then you will trigger FlushAll() when `flush_after_recovery` is `false`.